kosmorrolib/enum.py

```diff
@@ -25,7 +25,7 @@ class EventType(Enum):
     MAXIMAL_ELONGATION = 4
     MOON_PERIGEE = 5
     MOON_APOGEE = 6
-
+    EARTH_SEASON_CHANGE = 7
```

Comment:
I think we could instead add four distinct constants, one per season: `WINTER_SOLSTICE`, `SPRING_EQUINOX`, `SUMMER_SOLSTICE` and `AUTUMN_EQUINOX`.

What do you think?